guides/source/configuring.md

```diff
@@ -699,13 +705,27 @@ query to the database to get this information. Defaults to `true`.
 Indicates whether to use a stable `#cache_key` method that is accompanied by a
 changing version in the `#cache_version` method.
 
+The default value depends on the `config.load_defaults` target version:
+
+| Starting with version | The default value is |
+| --------------------- | -------------------- |
+|                       | `false`              |
+| 5.2                   | `true`               |
```

Comment:
The table rows are in chronological order, rather than the reverse chronological order seen in the "Results of config.load_defaults" section.  So, here, the baseline default is `false`, and `true` is the default for "5.2 and up".  *However*, if a new row were added to this table (i.e. if the default changed again), the 2nd row could no longer be "5.2 and up"; it would have to be "5.2 up to (but not including) X.Y".

My hope was that the table headers could be read in such a way to communicate this:

* "Starting with version [unspecified], the default value is `false`."
* "Starting with version 5.2, the default value is `true`."
* ...and so on.

I will try to come up with some alternatives.